src/installers/index.ts

```diff
@@ -62,3 +63,9 @@ export const buildPkgInstallerMap = (
     installer: envVariablesInstaller,
   },
 });
+
+export type CT3APackageJSON = PackageJson & {
```

Comment:
yea I think this should probably be moved. When I skim through this file I would wonder `wtf is this doing?`

Reply:
Yeah true